docs/en/metrics/metrics-installation.asciidoc

```diff
@@ -1,100 +1,104 @@
 [[install-metrics-monitoring]]
 [role="xpack"]
-== Install Metrics
+== Get started
 
-The easiest way to get started with Elastic Logs is by using our hosted {es} Service on Elastic Cloud.
-The {es} Service is available on both AWS and GCP, and automatically configures {es} and {kib}.
+To use the {metrics-app}, you need {es} for storing and searching your data, and {kib}
+(version 6.5 or later) for visualizing and managing it.
```

Comment:
I think we can drop mention of 6.6 now that we're more than half way through the 7.n release cycle.